2025-04-21 15:13:00,035 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.3.96:5000
2025-04-21 15:13:00,035 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-04-21 15:18:00,363 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided o8mb126224394wmp']
2025-04-21 16:17:57,327 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided m5mb120976123wrx']
2025-04-21 16:19:35,855 - werkzeug - INFO - 192.168.3.96 - - [21/Apr/2025 16:19:35] "[33mGET / HTTP/1.1[0m" 404 -
2025-04-21 16:19:36,001 - werkzeug - INFO - 192.168.3.96 - - [21/Apr/2025 16:19:36] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-04-21 16:22:57,862 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided y20mb82596185wmq']
2025-04-21 16:27:58,274 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided hg18mb126770945wmb']
2025-04-21 16:32:58,847 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided i5mb121767692wrp']
2025-04-21 16:37:59,275 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided m14mb121097657wrx']
2025-04-21 16:42:59,720 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided x18mb108762223wrp']
2025-04-21 16:57:35,164 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided t2mb121822890wrx']
2025-04-21 17:00:30,328 - werkzeug - INFO - 127.0.0.1 - - [21/Apr/2025 17:00:30] "[33mGET / HTTP/1.1[0m" 404 -
2025-04-21 17:00:30,439 - werkzeug - INFO - 127.0.0.1 - - [21/Apr/2025 17:00:30] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-04-21 17:00:56,851 - werkzeug - INFO - 192.168.3.96 - - [21/Apr/2025 17:00:56] "[33mGET / HTTP/1.1[0m" 404 -
2025-04-21 17:02:35,615 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided o2mb78654356wms']
2025-04-21 17:07:36,033 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided m41mb127279360wms']
2025-04-21 17:12:36,492 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided f11mb127064118wmq']
2025-04-21 17:17:36,971 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided r16mb121467170wrn']
2025-04-21 17:22:37,600 - __main__ - ERROR - Erreur lors de la vérification des e-mails: LOGIN command error: BAD [b'Too many arguments provided h19mb127760695wmp']
2025-04-21 17:25:18,176 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.3.96:5000
2025-04-21 17:25:18,176 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-04-21 17:35:48,352 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.3.96:5000
2025-04-21 17:35:48,352 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-04-21 17:58:23,177 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.3.96:5000
2025-04-21 17:58:23,177 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-04-21 18:03:23,388 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-21 18:09:10,429 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-21 18:28:56,625 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-21 19:54:26,780 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-21 19:54:26,897 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
2025-04-22 19:39:04,094 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-22 19:55:21,050 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-22 21:11:57,687 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-22 21:11:57,866 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
2025-04-23 09:07:50,332 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:12:50,976 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:17:51,331 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:22:51,645 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:27:52,034 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:32:52,458 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:37:52,948 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:42:53,383 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:47:53,767 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:52:54,094 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 09:57:54,437 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:02:54,782 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:18:13,998 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:23:14,486 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:24:46,570 - werkzeug - INFO - 127.0.0.1 - - [23/Apr/2025 10:24:46] "[33mGET / HTTP/1.1[0m" 404 -
2025-04-23 10:28:14,802 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:33:15,139 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:38:15,489 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:43:15,802 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 10:55:07,070 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:10:00,747 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:15:01,067 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:20:01,358 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:25:01,722 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:30:02,065 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:35:02,333 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:40:02,694 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 11:45:03,061 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:00:12,127 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:05:12,666 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:10:12,971 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:15:13,240 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:20:13,573 - __main__ - ERROR - Erreur lors de la vérification des e-mails: [Errno 11001] getaddrinfo failed
2025-04-23 12:36:25,785 - waitress - INFO - Serving on http://0.0.0.0:5000
2025-04-23 13:43:49,071 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.3.96:5000
2025-04-23 13:43:49,071 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-04-23 13:48:50,019 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 13:48:52,085 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F11454490>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 14:31:54,450 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 14:31:56,501 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F11478B50>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 14:53:19,954 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 14:53:21,975 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F11479A10>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 14:58:20,662 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 14:58:24,687 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F1147A790>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:03:22,416 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:03:27,476 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F1147B550>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:08:23,920 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:08:29,943 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F1147B350>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:13:25,002 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:18:25,792 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:23:26,947 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:28:28,269 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:33:29,373 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:38:30,064 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:43:31,461 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:43:34,501 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F11481110>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:48:32,556 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:48:36,580 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F11478FD0>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:53:33,686 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 15:53:38,753 - __main__ - ERROR - Erreur lors du traitement des mises à jour Redmine: HTTPConnectionPool(host='192.168.3.96', port=3000): Max retries exceeded with url: /issues.json?key=dbc5d3f317cf69e27f2715c91734a64e316ef429&updated_on=%3E%3D2000-01-01T00%3A00%3A00Z&sort=updated_on%3Adesc&limit=100&offset=0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000014F1147A550>: Failed to establish a new connection: [WinError 10061] Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée'))
2025-04-23 15:58:34,821 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:03:36,191 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:08:37,311 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:13:38,626 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:21:46,520 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:26:47,595 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:31:50,030 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:36:30,726 - werkzeug - INFO - 127.0.0.1 - - [23/Apr/2025 16:36:30] "[33mGET / HTTP/1.1[0m" 404 -
2025-04-23 16:36:50,732 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:41:52,053 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:46:52,876 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 16:52:03,500 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 17:05:25,785 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 17:10:26,682 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 17:15:27,847 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
2025-04-23 17:20:28,600 - __main__ - ERROR - Erreur lors de la vérification des e-mails: b'[ALERT] Application-specific password required: https://support.google.com/accounts/answer/185833 (Failure)'
